src/Volume.ts

```diff
@@ -200,14 +209,27 @@ export default class Volume {
     this.normRegionOffset = subregionOffset.clone().divide(volumeSize);
   }
 
+  updateRequiredData(required: Partial<LoadSpec>) {
```

Comment:
This made me go down a typescript rabbit hole haha

Reply:
I believe I first spotted this pattern in the signature for `setState` in React class components. The argument to that method has type `Partial<State>` in order to capture the idea that you can pass in an incomplete `State` object containing only fields which need updating. Same idea here.